tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -208,7 +208,7 @@
 
     <!-- Integration: System.ServiceModel -->
     <!-- Assembly: System.ServiceModel -->
-    <!-- Latest package https://www.nuget.org/packages/System.ServiceModel.Http/4.10.2 -->
+    <!-- Latest package https://www.nuget.org/packages/System.ServiceModel.Http/6.0.0 -->
```

Comment:
AFAICT, we don't _actually_ instrument System.ServiceModel.Http. It's just a "proxy" for the System.ServiceModel (WCF) integration we have, which is a .NET Framework-only integration.

The 6.0.0 version of System.ServiceModel.Http is .NET 6+ only and isn't _actually_ used in any integration tests, so I'm not sure there's much value in including it here? 🤔